server/src/main/java/org/candlepin/policy/ValidationResult.java

```diff
@@ -27,13 +28,15 @@
 
     private List<ValidationError> errors;
     private List<ValidationWarning> warnings;
+    private List<Enum> errorKeys;
```

Comment:
We could have our validation enums implement an interface here to keep it more limited than just any enum, and expose the buildErrorMessage method at this level.

Reply:
That is a good idea. I'll do that

Reply:
Done.